backend/src/test/java/com/bakdata/conquery/integration/common/LoadingUtil.java

```diff
@@ -210,12 +211,56 @@ public static void importConcepts(StandaloneSupport support, ArrayNode rawConcep
 		);
 
 		for (Concept<?> concept : concepts) {
-
 			support.getDatasetsProcessor().addConcept(dataset, concept);
 		}
 	}
 
 
+	private static List<Concept<?>> getConcepts(StandaloneSupport support, ArrayNode rawConcepts) throws JSONException, IOException {
+		return ConqueryTestSpec.parseSubTreeList(
+				support,
+				rawConcepts,
+				Concept.class,
+				c -> c.setDataset(support.getDataset())
+		);
+	}
+
+	public static void addConcepts(StandaloneSupport support, ArrayNode rawConcepts, @NonNull Response.Status.Family expectedResponseFamily, @NonNull String expectedReason)
+			throws JSONException, IOException {
+		List<Concept<?>> concepts = getConcepts(support, rawConcepts);
+		for (Concept<?> concept : concepts) {
+			addOrUpdateConcept(support, concept, expectedResponseFamily, expectedReason, false);
+		}
+	}
+
+	public static void updateConcepts(StandaloneSupport support, ArrayNode rawConcepts, @NonNull Response.Status.Family expectedResponseFamily, @NonNull String expectedReason)
+			throws JSONException, IOException {
+		List<Concept<?>> concepts = getConcepts(support, rawConcepts);
+		for (Concept<?> concept : concepts) {
+			addOrUpdateConcept(support, concept, expectedResponseFamily, expectedReason, true);
+		}
+	}
+
+	private static void addOrUpdateConcept(@NonNull StandaloneSupport support, @NonNull Concept<?> concept, @NonNull Response.Status.Family expectedResponseFamily, @NonNull String expectedReason, boolean update) {
+		final URI
+				conceptURI =
+				HierarchyHelper.hierarchicalPath(support.defaultAdminURIBuilder(), AdminDatasetResource.class, update ? "updateConcept" : "addConcept")
+							   .buildFromMap(Map.of(
+									   ResourceConstants.DATASET, support.getDataset().getId()
+							   ));
+		final Response response = support.getClient()
+										 .target(conceptURI)
+										 .request(MediaType.APPLICATION_JSON)
+										 .put(Entity.entity(concept, MediaType.APPLICATION_JSON_TYPE));
+		JsonNode responseEntity = response.readEntity(JsonNode.class);
+		if (responseEntity != null) {
+			log.info(String.format("addOrUpdateConcept-response-entity : %s", responseEntity.toPrettyString()));
+		}
```

Comment:
Das musst du eigentlich nicht loggen.

Und wenn, dann direkt log4j zum formatieren:

```suggestion
		if (responseEntity != null) {
			log.info("addOrUpdateConcept-response-entity : {}", responseEntity.toPrettyString());
		}
```